CMSgov/onemac
# add withdraw functionality to appendix k

Story: https://qmacbis.atlassian.net/browse/OY2-21189
Endpoint: See github-actions bot comment

### Details

Enable withdraw of app k.

### Changes

- added action and config for lambda in serverless.yml
- Modified actionApi to match naming
- fixed issue in updateComponent that was causing withdraw email failures

### Test Plan

1. Login as a statesubmitter
2. Submit or locate an existing app k
3. navigate to the package dashboard and select withdraw package from the actions column for that app k
4. verify the popup has correct language and select yes, withdraw package
5. verify the package dashboard refreshes with the green success banner and the appk is now in withdrawn status
